### PR TITLE
Added ouia ids to GlobalFilter

### PR DESCRIPTION
**Pull request by computercamplove**
Added ouia ids to GlobalFilter, GlobalFilterMenu and TagsModal.

cc @karelhala

### CODE REVIEW

File: src/js/App/GlobalFilter/GlobalFilterMenu.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ const GlobalFilterMenu = (props) => {
                     !!Object.values(selectedTags).find((group = {}) => group[tagKey]?.isSelected);
                   return (
                     <MenuItem key={value} onClick={onClick}>
-                      <Checkbox className="ins-c-global-filter__checkbox" id={id} isChecked={isChecked} label={label} />
+                      <Checkbox className="ins-c-global-filter__checkbox" ouiaId="filter-checkbox" id={id} isChecked={isChecked} label={label} />
```
Comment (jjaquish) [inline src/js/App/GlobalFilter/GlobalFilterMenu.js:112]: I would change this one to `global-filter-checkbox`

File: src/js/App/GlobalFilter/GlobalFilter.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ const GlobalFilterDropdown = ({
                   </ChipGroup>
                 ))}
                 {!isDisabled && (
-                  <Button variant="link" onClick={() => setValue(() => ({}))}>
+                  <Button variant="link" ouiaId="clear-global-filters" onClick={() => setValue(() => ({}))}>
```
Comment (jjaquish) [inline src/js/App/GlobalFilter/GlobalFilter.js:77]: I think I would try to make all of these start the same for consistency and contain global-filter in some way. Here maybe `global-filter-clear`

File: src/js/App/GlobalFilter/GlobalFilterMenu.js [hunk 3]
```diff
@@ -134,6 +134,7 @@ const GlobalFilterMenu = (props) => {
       className={classNames('ins-c-global-filter__select', {
         expanded: isOpen,
       })}
+      ouiaId="select-global-filter"
```
Comment (jjaquish) [inline src/js/App/GlobalFilter/GlobalFilterMenu.js:137]: potential OUIA ID: `global-filter-select`

File: src/js/App/GlobalFilter/GlobalFilterMenu.js [hunk 4]
```diff
@@ -146,6 +147,7 @@ const GlobalFilterMenu = (props) => {
           onChange={onFilter}
           placeholder="Filter by status"
           aria-label="Filter by status"
+          ouiaId="filter-by-status"
```
Comment (jjaquish) [inline src/js/App/GlobalFilter/GlobalFilterMenu.js:150]: potential OUIA ID: `global-filter-filter-by-status` or `global-filter-by-status`

File: src/js/App/GlobalFilter/TagsModal.js [hunk 5]
```diff
@@ -182,6 +182,7 @@ const TagsModal = ({ isOpen, filterTagsBy, onApplyTags, toggleModal, selectedTag
       selected={[tagsSelected, sidsSelected]}
       onApply={() => onApplyTags(tagsSelected, sidsSelected)}
       title="Select one or more tags/SAP IDs (SID)"
+      ouiaId="filter-tags-modal"
```
Comment (jjaquish) [inline src/js/App/GlobalFilter/TagsModal.js:185]: Assuming this modal is only used by the global filter, probably `global-filter-tags-modal` for the OUIA

**Review by jjaquish — COMMENTED** [review]
Looks good as far as I can tell, just left a comment about changing some OUIA IDs for the sake of consistency

**Review by jjaquish — COMMENTED** [review]
LGTM

**Review by Hyperkid123 — APPROVED** [review]

**Review by karelhala — APPROVED** [review]

**Pull request merged**